openml/datasets/dataset.py

```diff
@@ -86,23 +84,23 @@ class OpenMLDataset(object):
     dataset: string, optional
         Serialized arff dataset string.
     """
-    def __init__(self, name, description, format, dataset_id=None,
+    def __init__(self, name, description, data_format='arff', dataset_id=None,
```

Comment:
Do we care about backward compatibility? Changing the name would require a deprecation warning.